icicle/backend/cpu/src/curve/test/msm_test.cpp

```diff
@@ -6,7 +6,7 @@
 #include <cassert>
 #include "timer.hpp"
 
-// #define DUMMY_TYPES
+#define DUMMY_TYPES
```

Comment:
didn't you want to remove this test? I don't know if you should but you wanted I think